pages/card.html

```diff
@@ -0,0 +1,349 @@
+            sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <div class="badge_container-main flex-center">
+            <section>
+                <div class="card-container">
+                    <div class="image-div">
+                        <div class="overlay-image">
+                            <img class="card-image" src="../images/card.jpg" alt="error" />
```

Comment:
can give specific alt attribute error is not that clear if  by chance image not loaded

Reply:
OK!